src/test/java/ProjectOcean/Model/CoursePlanningSystemTests.java

```diff
@@ -151,5 +151,15 @@ public void executeSearchTest() {
                 Assert.assertTrue(course.getCourseName().toLowerCase().contains("maskin") || course.getCourseName().toLowerCase().contains("matematisk"));
             }
         }
+
+        //tests searching for course type
+        searchText = "Naturvetenskap";
+        searchResult = model.executeSearch(searchText);
+        Assert.assertFalse(searchResult.isEmpty());
+        if(!searchResult.isEmpty()) {
```

Comment:
Not necessary if-statement, since row 158 will throw error if researchResult is empty.